stylesheets/_component.buttons.scss

```diff
@@ -1,385 +1,110 @@
-    &.is-disabled,
-    &.disabled {
+    &.disabled,
+    &.is-disabled {
         opacity: .5;
-
-        &.btn--naked {
-            color: color($state, dark) !important;
-
-            &:active,
-            &.active {
-                background-color: transparent !important;
-                color: color($state, dark) !important;
-            }
-        }
-    }
-
-    > .badge {
-        background-color: color($state, dark);
-        color: pick_best_color(color($state, dark), (#fff, color(gray, dark))) !important;
-
-        &.badge--outline {
-            background-color: transparent;
-            color: inherit !important;
-        }
-    }
-
-    &.btn--naked {
-        color: color($state, dark) !important;
-
-        &:active,
-        &.active {
-            background-color: color($state, light) !important;
-            color: color($state, dark) !important;
-
-            &:not(:disabled):not(.is-disabled).btn--#{$btn-white} {
-                color: color(text) !important;
-            }
-        }
-    }
-}
-
-/**
- * UI states
- */
-
-.btn {
-    @include button-states('base');
-    border-bottom-style: solid;
-    text-decoration: none;
-
-    &:disabled,
-    &.is-disabled,
-    &.disabled {
-        cursor: not-allowed !important;
+        pointer-events: none;
     }
 }
 
+// State variations
 @each $state, $state-color, $state-color-alt in $state-colors {
     .btn--#{'' + $state} {
-        @include button-states($state);
+        @include button-variant($state);
     }
 }
 
+// Outline button styles
 .btn--outline {
-    background: none !important;
-    border: 2px solid color(base);
-    box-shadow: none;
-    padding: ($btn-padding-top - 2) ($btn-padding-x - 2) $btn-padding-bottom;
-
-    &:hover,
-    &.is-hovered {
-        box-shadow: none;
-        margin-bottom: 1px; // offsets 1px padding reduction that causes content below to jump on button hover
-        padding-bottom: $btn-padding-bottom - 1;
-
-        &:focus {
-            box-shadow: 0 1px 0 color(black);
-        }
-
-        &:focus:active {
-            box-shadow: none;
-        }
-    }
-
-    &:focus {
-        @include pulsar-outline-button-focused;
-    }
-
-    &:active,
-    &.active {
-        background-color: color(base, light) !important;
-        box-shadow: none;
-        color: pick_best_color(color(base, light), (#fff, color(gray, dark))) !important;
-        padding-bottom: $btn-padding-bottom - 2;
-        top: 2px;
-    }
-
-    &:active:focus {
-        border: 2px solid color(base);
-    }
-
-    &.is-disabled,
-    &:disabled {
-        background-color: transparent !important;
-        box-shadow: none !important;
-    }
-
-    &.btn--naked {
-        padding: $btn-padding-top $btn-padding-x $btn-padding-bottom !important;
-    }
-
-    &.btn--small {
-        padding: ($btn-small-padding-top - 2) ($btn-small-padding-x - 2) ($btn-small-padding-bottom - 2) !important;
+    background: none;
+    border: 1px solid color(base);
```

Comment:
See notes on this. Feels like 1px is too thin.

Reply:
I'm going to run with it for now, most notable design systems use 1px for outline buttons and I'm hoping this will reduce our reliance on naked buttons a little by being subtle enough to not draw too much focus

Reply:
Leaving for now